src/components/UserSelector.tsx

```diff
@@ -1,24 +1,22 @@
-import React, { useContext, useEffect, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import classNames from 'classnames';
-import { UserContext } from './UsersContext';
+import { useAppDispatch, useAppSelector } from '../app/hooks';
 import { User } from '../types/User';
```

Comment:
Ensure that the `User` type is correctly defined and imported from the `../types/User` file. Verify its existence and correctness.